services/press-fault-detection-model-service/app/routers/predict_router.py

```
@@ -0,0 +1,24 @@
+)
+
+@router.post("", response_model=PredictionResponse)
+async def predict_fault(request_body: SensorData):
+    
+    try:
+        result = predict(data=request_body)
+        return PredictionResponse(**result)
+    
+    except RuntimeError as e:
+        # 모델이 로드되지 않았을 경우
+        raise HTTPException(status_code=503, detail=str(e))
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)